pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -107,6 +110,16 @@ func (m *Multicluster) AddMemberCluster(clientset kubernetes.Interface, metadata
 	_ = kubectl.AppendServiceHandler(func(svc *model.Service, ev model.Event) { m.updateHandler(svc) })
 	_ = kubectl.AppendInstanceHandler(func(si *model.ServiceInstance, ev model.Event) { m.updateHandler(si.Service) })
 	go kubectl.Run(stopCh)
+	opts := Options{
+		ResyncPeriod: m.ResyncPeriod,
+		DomainSuffix: m.DomainSuffix,
+	}
+	nc := NewNamespaceController(func() map[string]string {
+		return map[string]string{
+			constants.CACertNamespaceConfigMapDataName: m.caroot,
```

Comment:
This won't support the root cert changing like the non-multicluster variant does. Is that ok?

Reply:
I think it expires in 10 years or so - and a restart of istiod in the next 10 years will handle this.